framework-plugins/lisk-framework-report-misbehavior-plugin/src/endpoint.ts

```diff
@@ -37,22 +37,23 @@ export class Endpoint extends BasePluginEndpoint {
 
 	// eslint-disable-next-line @typescript-eslint/require-await
 	public async authorize(context: PluginEndpointContext): Promise<{ result: string }> {
-		validator.validate(actionParamsSchema, context.params);
+		validator.validate<{ enable: boolean; password: string }>(actionParamsSchema, context.params);
 
 		const { enable, password } = context.params;
 
 		try {
 			const parsedEncryptedPassphrase = encrypt.parseEncryptedMessage(
-				this._config.encryptedPassphrase,
+				this._config.encryptedPrivateKey,
 			);
 
-			const passphrase = await encrypt.decryptMessageWithPassword(
-				parsedEncryptedPassphrase as any,
-				password as string,
+			const privateKeyStr = await encrypt.decryptMessageWithPassword(
+				parsedEncryptedPassphrase,
```

Comment:
To complete the refactoring, the variable `parsedEncryptedPassphrase` should be renamed into `parsedEncryptedPrivateKey`.